crypto/rand/rand_egd.c

```diff
@@ -100,6 +158,18 @@ int RAND_query_egd_bytes(const char *path, unsigned char *buf, int bytes)
             /* No error, try again */
             break;
         default:
+# if defined(OPENSSL_SYSNAME_TANDEM)
+              if (hpns_connect_attempt == 0)
+              {
+                /* try the other kind of AF_UNIX socket */
+                close(fd);
+                fd = hpns_socket(AF_UNIX, SOCK_STREAM, 0, AF_UNIX_PORTABILITY);
+                if (fd == -1) return (-1);
```

Comment:
Nit: No parens for return, i.e. just `return -1`.

Reply:
and the return goes on its own line, n'est ce pas ?

Reply:
Nit fixed.

The original effort to port 1.1.1 was done (and shared) as part of #5087 for 1.1.1. The 3.0.0 port is 100% my effort alone.